activation/atxdb.go

```diff
@@ -696,9 +706,7 @@ func (db *DB) HandleAtxData(ctx context.Context, data []byte) error {
 	existing, _ := db.GetAtxHeader(atx.ID())
```

Comment:
what do you think about adding a HasATX() interface and just do a key lookup instead of loading the header

Reply:
makes sense, probably simpler to add that after (or together with) sql changes